ttnn/cpp/ttnn/operations/data_movement/repeat/repeat.cpp

```diff
@@ -18,12 +19,22 @@ ttnn::Tensor RepeatOperation::invoke(
     const Shape & repeat_dims,
     const std::optional<MemoryConfig>& memory_config_arg) {
 
+    ttnn::Tensor interleaved_input_tensor;
+    if(input_tensor.is_sharded()) {
+        auto dram_memory_config = tt::tt_metal::MemoryConfig{.memory_layout=tt::tt_metal::TensorMemoryLayout::INTERLEAVED,.buffer_type=tt::tt_metal::BufferType::DRAM};
+        interleaved_input_tensor = ttnn::sharded_to_interleaved(queue_id, input_tensor, dram_memory_config, std::nullopt);
+    }
+    else {
+        interleaved_input_tensor = input_tensor;
+    }
```

Comment:
Does it mean that the op only supported interleaved but did not check its input?